config/routes.rb

```diff
@@ -26,6 +26,13 @@
         end
       end
       resources :room_settings, only: %i[update], param: :friendly_id
+      resources :shared_accesses, only: :destroy, path: '/shared_accesses/room', param: :friendly_id do
+        member do
+          post :create
```

Comment:
We need to scope this with friendly_id because it is needed as a params in the POST/create request.
Could also have a hidden_field with the friendly_id in the Create form.

Reply:
It will be scoped if you add it to `only: %i[create destory]`